engine/Build.bee.cs

```diff
@@ -223,12 +223,22 @@ static void DeployIOS()
 
     static void Main()
     {
-        flutterRoot = Environment.GetEnvironmentVariable("FLUTTER_ROOT_PATH");
-        if (string.IsNullOrEmpty(flutterRoot))
+        preBuildLibRoot = Environment.GetEnvironmentVariable("PREBUILD_LIB_ROOT_PATH");
```

Comment:
Personally speaking, the different between `build from flutter engine source` and `build from pre-build library` is only how we get the libraries, therefore doesn't affect the build process of the uiwidget library (i.e., the codes here in the `Build.bee.cs` file)

As the result, I prefer to not change any build code here. Instead, we can use the original  `FLUTTER_ROOT_PATH` but only put the required libraries, header files in the proper sub folders.

Reply:
Updated